ph5/utilities/tests/test_segd2ph5.py

```diff
@@ -349,8 +349,8 @@ def test_main(self):
         self.assertEqual(self.ph5object.Array_t_names, ['Array_t_001'])
         self.ph5object.read_array_t('Array_t_001')
         a = self.ph5object.Array_t['Array_t_001']['byid']['1'][2][0]
-        self.assertAlmostEqual(a['location/Y/value_d'], 19.421, 2)
-        self.assertAlmostEqual(a['location/X/value_d'], -155.291, 2)
+        self.assertEqual(a['location/Y/value_d'], 19.42083740234375)
```

Comment:
Are we sure that we want this much precision in the latitude and longitude? The SEED manual ha a precision of 6 decimal places, so the output looks like ###.######. It seems like we should be consistent with the seed manual.

Reply:
I mentioned in the issue that 6 (SEED) or 7 (I get PI-surveyed coordinates like this) decimal places would be workable, but also now I'm wondering: Do we want the original coordinates as read from the SEG-D headers preserved in the PH5 metadata?  That seems like it might be useful if troubleshooting is needed in the future, and in the meantime the end user still sees the 6 decimal places SEED convention.

If the SmartSolo coordinates get rounded to 6 decimal places during ingestion, then I think the Fairfield coordinates also need to get rounded for consistency.

Reply:
We should not loose data if we can help it. What is the precision in the SmartSolo headers?

Reply:
I am not aware of a limit for either node manufacturer, but I think it usually goes out to 13-14 decimal places.  If the array table is updated with PI-provided coordinates, then that level of precision probably is no longer in the PH5 volume, but the deleted array table(s) would still be available from the backup kef.  (I guess it'd be theoretically possible to feed the coordinates as read from the headers into a new, non-array table so they'd still be there even in that case, but that seems like overkill.)

Reply:
It seems like we should probably use 6 to stay consistent with SEED conventions. 13 to 14 latitude and longitude decimal places are likely an artifact created by some internal smart solo calculations, and are not precise observations.  https://gis.stackexchange.com/questions/8650/measuring-accuracy-of-latitude-and-longitude